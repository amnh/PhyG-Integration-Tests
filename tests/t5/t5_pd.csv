Alentus,Amblypygid,Americhernus,Aportus,Artemia,Centruoides,Chanbria,Gea,Hadrurus,Hypochilus,Limulus,Mastigoproctus,Pauroctonus,Rhiphicephalus,Thelichoris,Thermobius,Vonones
"0.0","48.0","76.0","23.0","70.0","59.0","61.0","69.0","58.0","53.0","54.0","57.0","57.0","59.0","56.0","58.0","48.0"
"48.0","0.0","68.1025641025641","51.30909090909091","62.25000000000001","44.980645161290326","41.5","60.65384615384615","51.406451612903226","31.058064516129033","39.6258064516129","38.30769230769231","47.12258064516129","53.58227848101266","28.916129032258066","54.99386503067484","35.94904458598726"
"76.0","68.1025641025641","0.0","75.45454545454545","77.8125","74.48717948717949","77.67948717948718","75.55128205128204","72.35897435897436","73.42307692307692","72.35897435897436","76.61538461538461","71.2948717948718","76.69620253164557","70.23076923076923","73.32515337423312","69.78343949044586"
"23.0","51.30909090909091","75.45454545454545","0.0","70.42424242424242","58.35151515151515","60.36363636363637","69.41818181818182","62.375757575757575","59.35757575757576","55.333333333333336","55.333333333333336","60.36363636363637","63.38181818181818","56.33939393939394","61.369696969696975","52.31515151515152"
"70.0","62.25000000000001","77.8125","70.42424242424242","0.0","57.06250000000001","56.025000000000006","64.325","61.212500000000006","57.06250000000001","58.10000000000001","53.95","58.10000000000001","64.325","57.06250000000001","57.03067484662576","58.10000000000001"
"59.0","44.980645161290326","74.48717948717949","58.35151515151515","57.06250000000001","0.0","46.82051282051282","59.58974358974359","46.05161290322581","42.83870967741936","40.96103896103896","39.37179487179487","38.80519480519481","58.835443037974684","42.83870967741936","54.99386503067484","48.63694267515923"
"61.0","41.5","77.67948717948718","60.36363636363637","56.025000000000006","46.82051282051282","0.0","63.84615384615385","58.52564102564102","39.37179487179487","34.05128205128205","36.17948717948718","50.01282051282051","58.835443037974684","40.43589743589744","58.04907975460122","43.35031847133758"
"69.0","60.65384615384615","75.55128205128204","69.41818181818182","64.325","59.58974358974359","63.84615384615385","0.0","58.52564102564102","46.82051282051282","57.46153846153846","55.333333333333336","53.205128205128204","69.34177215189874","47.88461538461539","63.141104294478524","60.267515923566876"
"58.0","51.406451612903226","72.35897435897436","62.375757575757575","61.212500000000006","46.05161290322581","58.52564102564102","58.52564102564102","0.0","50.33548387096774","51.406451612903226","51.07692307692308","25.703225806451613","59.88607594936709","49.26451612903226","56.0122699386503","52.86624203821656"
"53.0","31.058064516129033","73.42307692307692","59.35757575757576","57.06250000000001","42.83870967741936","39.37179487179487","46.82051282051282","50.33548387096774","0.0","35.34193548387097","32.98717948717949","42.83870967741936","54.63291139240506","9.638709677419355","53.97546012269938","39.12101910828025"
"54.0","39.6258064516129","72.35897435897436","55.333333333333336","58.10000000000001","40.96103896103896","34.05128205128205","57.46153846153846","51.406451612903226","35.34193548387097","0.0","39.37179487179487","42.03896103896104","53.58227848101266","39.6258064516129","52.95705521472392","35.94904458598726"
"57.0","38.30769230769231","76.61538461538461","55.333333333333336","53.95","39.37179487179487","36.17948717948718","55.333333333333336","51.07692307692308","32.98717948717949","39.37179487179487","0.0","44.69230769230769","51.48101265822785","36.17948717948718","56.0122699386503","45.46496815286624"
"57.0","47.12258064516129","71.2948717948718","60.36363636363637","58.10000000000001","38.80519480519481","50.01282051282051","53.205128205128204","25.703225806451613","42.83870967741936","42.03896103896104","44.69230769230769","0.0","56.734177215189874","41.767741935483876","54.99386503067484","46.52229299363057"
"59.0","53.58227848101266","76.69620253164557","63.38181818181818","64.325","58.835443037974684","58.835443037974684","69.34177215189874","59.88607594936709","54.63291139240506","53.58227848101266","51.48101265822785","56.734177215189874","0.0","57.78481012658228","56.0122699386503","52.53164556962025"
"56.0","28.916129032258066","70.23076923076923","56.33939393939394","57.06250000000001","42.83870967741936","40.43589743589744","47.88461538461539","49.26451612903226","9.638709677419355","39.6258064516129","36.17948717948718","41.767741935483876","57.78481012658228","0.0","58.04907975460122","42.29299363057325"
"58.0","54.99386503067484","73.32515337423312","61.369696969696975","57.03067484662576","54.99386503067484","58.04907975460122","63.141104294478524","56.0122699386503","53.97546012269938","52.95705521472392","56.0122699386503","54.99386503067484","56.0122699386503","58.04907975460122","0.0","51.938650306748464"
"48.0","35.94904458598726","69.78343949044586","52.31515151515152","58.10000000000001","48.63694267515923","43.35031847133758","60.267515923566876","52.86624203821656","39.12101910828025","35.94904458598726","45.46496815286624","46.52229299363057","52.53164556962025","42.29299363057325","51.938650306748464","0.0"
