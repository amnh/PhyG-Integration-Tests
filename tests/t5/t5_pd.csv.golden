Alentus,Amblypygid,Americhernus,Aportus,Artemia,Centruoides,Chanbria,Gea,Hadrurus,Hypochilus,Limulus,Mastigoproctus,Pauroctonus,Rhiphicephalus,Thelichoris,Thermobius,Vonones
"0.0","48.0","76.0","23.0","70.0","59.0","61.0","69.0","58.0","53.0","54.0","57.0","57.0","59.0","56.0","58.0","48.0"
"48.0","0.0","64.0","51.0","60.0","42.0","39.0","57.0","48.0","29.0","37.0","36.0","44.0","51.0","27.0","54.0","34.0"
"76.0","64.0","0.0","75.0","75.0","70.0","73.0","71.0","68.0","69.0","68.0","72.0","67.0","73.0","66.0","72.0","66.0"
"23.0","51.0","75.0","0.0","70.0","58.0","60.0","69.0","62.0","59.0","55.0","55.0","60.0","63.0","56.0","61.0","52.0"
"70.0","60.0","75.0","70.0","0.0","55.0","54.0","62.0","59.0","55.0","56.0","52.0","56.0","62.0","55.0","56.0","56.0"
"59.0","42.0","70.0","58.0","55.0","0.0","44.0","56.0","43.0","40.0","38.0","37.0","36.0","56.0","40.0","54.0","46.0"
"61.0","39.0","73.0","60.0","54.0","44.0","0.0","60.0","55.0","37.0","32.0","34.0","47.0","56.0","38.0","57.0","41.0"
"69.0","57.0","71.0","69.0","62.0","56.0","60.0","0.0","55.0","44.0","54.0","52.0","50.0","66.0","45.0","62.0","57.0"
"58.0","48.0","68.0","62.0","59.0","43.0","55.0","55.0","0.0","47.0","48.0","48.0","24.0","57.0","46.0","55.0","50.0"
"53.0","29.0","69.0","59.0","55.0","40.0","37.0","44.0","47.0","0.0","33.0","31.0","40.0","52.0","9.0","53.0","37.0"
"54.0","37.0","68.0","55.0","56.0","38.0","32.0","54.0","48.0","33.0","0.0","37.0","39.0","51.0","37.0","52.0","34.0"
"57.0","36.0","72.0","55.0","52.0","37.0","34.0","52.0","48.0","31.0","37.0","0.0","42.0","49.0","34.0","55.0","43.0"
"57.0","44.0","67.0","60.0","56.0","36.0","47.0","50.0","24.0","40.0","39.0","42.0","0.0","54.0","39.0","54.0","44.0"
"59.0","51.0","73.0","63.0","62.0","56.0","56.0","66.0","57.0","52.0","51.0","49.0","54.0","0.0","55.0","55.0","50.0"
"56.0","27.0","66.0","56.0","55.0","40.0","38.0","45.0","46.0","9.0","37.0","34.0","39.0","55.0","0.0","57.0","40.0"
"58.0","54.0","72.0","61.0","56.0","54.0","57.0","62.0","55.0","53.0","52.0","55.0","54.0","55.0","57.0","0.0","51.0"
"48.0","34.0","66.0","52.0","56.0","46.0","41.0","57.0","50.0","37.0","34.0","43.0","44.0","50.0","40.0","51.0","0.0"
